### PR TITLE
Review and update four TiKV docs

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed?

I've reviewed and updated the following four TiKV docs.

- CAS on RawKV
- reference/architecture/overview.md
- reference/CLI/introduction.md
- Monitor and Alert

### CODE REVIEW

File: content/docs/5.1/concepts/explore-tikv-features/cas.md [hunk 1]
```diff
@@ -19,17 +21,19 @@ if (prevValue == request.prevValue) {
 return prevValue;
 ```
 
-The atomicity guarantees that the new value is calculated based on up-to-date information; if the value had been updated by another thread in the meantime, the write would fail.
+The atomicity guarantees that the new value is calculated based on the up-to-date information. If the value is updated by another thread at the same time, the write would fail.
 
 ## Prerequisites
 
-Please install TiUP, jshell, download tikv-client jars and start a TiKV Cluster according to [TiKV in 5 Minutes](../../tikv-in-5-minutes).
+Make sure that you have installed TiUP, jshell, download tikv-client JAR files，and start a TiKV cluster according to [TiKV in 5 Minutes](../../tikv-in-5-minutes).
```
Comment (CharLotteiu) [inline content/docs/5.1/concepts/explore-tikv-features/cas.md:28]:
```suggestion
Make sure that you have installed TiUP, jshell, download tikv-client JAR files, and start a TiKV cluster according to [TiKV in 5 Minutes](../../tikv-in-5-minutes).
```

File: content/docs/5.1/reference/CLI/introduction.md [hunk 2]
```diff
@@ -1,16 +1,14 @@
 ---
 title: CLI
-description: Command-Line Interface Tools used to administrate TiKV Cluster
+description: Command-line interface tools used to administrate TiKV Clusters
```
Comment (CharLotteiu) [inline content/docs/5.1/reference/CLI/introduction.md:3]:
```suggestion
description: Command-line interface tools used to administrate TiKV clusters
```

File: content/docs/5.1/reference/architecture/overview.md [hunk 3]
```diff
@@ -7,45 +7,49 @@ menu:
...
-The overall architecture of TiKV is illustrated below:
+The overall architecture of TiKV is as follows:
 
 {{< figure
     src="/img/tikv-architecture.png"
     caption="The architecture of TiKV"
     alt="TiKV architecture diagram" >}}
 
-A TiKV cluster consists of 2 components:
+A TiKV cluster consists of the following components:
```
Comment (CharLotteiu) [inline content/docs/5.1/reference/architecture/overview.md:21]: 个人觉得此处说 A TiKV cluster consists of a TiKV cluster and a PD cluster 读起来稍微有点逻辑问题，能否说 A TiKV cluster consists of a group of TiKV nodes and a PD cluster? 或者把第一个 TiKV cluster 换一个表述？ 比如 A complete TiKV cluster consists of ... 之类的。@marsishandsome PTAL.

File: content/docs/5.1/reference/architecture/overview.md [hunk 4]
```diff
@@ -7,45 +7,49 @@ menu:
...
 
-1. [A TiKV Cluster](./#tikv-cluster), which is responsible for storing key-value pair data.
-2. [A Placement Driver (PD) Cluster](./#pd-cluster), which works as the manager in a TiKV cluster.
+- [A TiKV cluster](./#tikv-cluster) storing key-value pair data
+- [A Placement Driver (PD) cluster](./#pd-cluster) working as the manager of a TiKV cluster
```
Comment (CharLotteiu) [inline content/docs/5.1/reference/architecture/overview.md:24]:
```suggestion
- [A Placement Driver (PD) cluster](./#pd-cluster) working as the manager of the TiKV cluster
```

File: content/docs/5.1/reference/architecture/overview.md [hunk 5]
```diff
@@ -7,45 +7,49 @@ menu:
...
+- [Timestamp oracle](/deep-dive/distributed-transaction/timestamp-oracle/)
 
-[Timestamp oracle](/deep-dive/distributed-transaction/timestamp-oracle/) plays a significant role in the Percolator Transaction model. PD implements such a service that hands out timestamps in strictly increasing order, which is a property required for the correct operation of the snapshot isolation protocol.
+   Timestamp oracle plays a significant role in the Percolator Transaction model. PD implements a service to hand out timestamps in the strictly increasing order, which is a property required for the correct operations of the snapshot isolation protocol.
 
-### Region Scheduler
+- Region Scheduler
 
-Data in TiKV is organized as Regions, which are replicated on several stores. Someone needs to be responsible for deciding the storage location of each replica, which is the second job of PD.
+    Data in TiKV is organized as Regions, which are replicated to several stores. Region Scheduler decides the storage location of each replica.
```
Comment (CharLotteiu) [inline content/docs/5.1/reference/architecture/overview.md:55]:
```suggestion
    Data in TiKV is organized as Regions, which are replicated to several stores. PD, as the Region scheduler, decides the storage location of each replica.
```

File: content/docs/5.1/reference/architecture/overview.md [hunk 6]
```diff
@@ -7,45 +7,49 @@ menu:
...
+As the manager in a TiKV cluster, the Placement Driver ([PD](https://github.com/tikv/pd)) provides the following functions:
 
-### Timestamp Oracle
+- [Timestamp oracle](/deep-dive/distributed-transaction/timestamp-oracle/)
 
-[Timestamp oracle](/deep-dive/distributed-transaction/timestamp-oracle/) plays a significant role in the Percolator Transaction model. PD implements such a service that hands out timestamps in strictly increasing order, which is a property required for the correct operation of the snapshot isolation protocol.
+   Timestamp oracle plays a significant role in the Percolator Transaction model. PD implements a service to hand out timestamps in the strictly increasing order, which is a property required for the correct operations of the snapshot isolation protocol.
 
-### Region Scheduler
+- Region Scheduler
```
Comment (CharLotteiu) [inline content/docs/5.1/reference/architecture/overview.md:53]:
```suggestion
- Region scheduler
```

File: content/docs/5.1/reference/architecture/overview.md [hunk 7]
```diff
@@ -7,45 +7,49 @@ menu:
...
+## PD cluster
 
-As the manager in a TiKV cluster, the Placement Driver ([PD](https://github.com/tikv/pd)) provides two major functions: [Timestamp Oracle](./#timestamp-oracle) and [Region Scheduler](./#region-scheduler).
+As the manager in a TiKV cluster, the Placement Driver ([PD](https://github.com/tikv/pd)) provides the following functions:
 
-### Timestamp Oracle
+- [Timestamp oracle](/deep-dive/distributed-transaction/timestamp-oracle/)
 
-[Timestamp oracle](/deep-dive/distributed-transaction/timestamp-oracle/) plays a significant role in the Percolator Transaction model. PD implements such a service that hands out timestamps in strictly increasing order, which is a property required for the correct operation of the snapshot isolation protocol.
+   Timestamp oracle plays a significant role in the Percolator Transaction model. PD implements a service to hand out timestamps in the strictly increasing order, which is a property required for the correct operations of the snapshot isolation protocol.
```
Comment (CharLotteiu) [inline content/docs/5.1/reference/architecture/overview.md:51]:
```suggestion
   Timestamp oracle plays a significant role in the Percolator transaction model. PD implements a service to hand out timestamps in the strictly increasing order, which is a property required for the correct operations of the snapshot isolation protocol.
```

File: content/docs/5.1/reference/architecture/overview.md (hunk 3 above)
Comment (marsishandsome) [reply content/docs/5.1/reference/architecture/overview.md:21]: 确实，改成 `a group of TiKV nodes` 会更好些。

**Review by marsishandsome — APPROVED** [review]
LGTM

**Comment by qiancai** [comment]
@CharLotteiu Thanks for the careful review and constructive comments. All the comments are accepted. PTAL again.

File: content/docs/5.1/reference/architecture/overview.md [hunk 8]
```diff
@@ -7,45 +7,49 @@ menu:
...
     caption="The architecture of TiKV"
     alt="TiKV architecture diagram" >}}
 
-A TiKV cluster consists of 2 components:
+A TiKV cluster consists of the following components:
 
-1. [A TiKV Cluster](./#tikv-cluster), which is responsible for storing key-value pair data.
-2. [A Placement Driver (PD) Cluster](./#pd-cluster), which works as the manager in a TiKV cluster.
+- [A group of TiKV nodes](./#tikv-cluster) storing key-value pair data
+- [A Placement Driver (PD) node](./#pd-cluster) working as the manager of the TiKV cluster
```
Comment (CharLotteiu) [inline content/docs/5.1/reference/architecture/overview.md:24]:
```suggestion
- [A Placement Driver (PD) cluster](./#pd-cluster) working as the manager of the TiKV cluster
```
PD 应该也有可能会部署在多个节点上，`a PD node` 有点不够准确，感觉还是 `cluster` 更合适一些。

@marsishandsome Please take a look~

**Review by CharLotteiu — APPROVED** [review]
Rest LGTM

File: content/docs/5.1/reference/architecture/overview.md (hunk 8 above)
Comment (marsishandsome) [reply content/docs/5.1/reference/architecture/overview.md:24]: LGTM

**Pull request merged**